services/src/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java

```diff
@@ -53,6 +54,9 @@
     @Autowired
     NorthboundService northbound;
 
+    @Value("#{'${kafka.bootstrap.server}'.split(':')}")
+    private List<String> kafkaBootstrapServer;
```

Comment:
Why `kafkaBootstrapServer` is a list of strings? this is weird

Reply:
Because it's should be server**s**